x-pack/plugins/security_solution/cypress/e2e/detection_response/rule_management/rules_table/rules_table_persistent_state.cy.ts

```diff
@@ -109,7 +111,7 @@ describe('Rules table: persistent state', () => {
     resetRulesTableState();
   });
 
-  describe('while on a happy path', () => {
+  describe('while on a happy path', { tags: [tag.ESS, tag.SERVERLESS] }, () => {
```

Comment:
Probably you missed it, but there's a top-level `describe` block at line 103. Let's tag it instead of tagging every nested `describe` block.